src/js/remoteServices.js

```diff
@@ -108,7 +108,8 @@ export default {
         document
           .querySelector('[data-testid="issue.views.issue-base.foundation.summary.heading"]')
           ?.textContent?.trim() ||
-        document.querySelector(".ghx-selected .ghx-summary")?.textContent?.trim()
+        document.querySelector(".ghx-selected .ghx-summary")?.textContent?.trim() ||
+        document.querySelector('[id="summary-val"]')?.innerText?.trim()
```

Comment:
Can you please indicate where this selector is shown in Jira?

Reply:
Hello Manuel,

yes, sure!

Here is the screenshot:
![image](https://github.com/user-attachments/assets/f55c7731-1ed1-4849-a177-246c87b0bec0)